certificator/__main__.py

```diff
@@ -1,15 +1,41 @@
 import click
+import os
 
 from .meetup import MeetupCertificator
+from .certificator import CSVCertificator
 
 
-@click.command()
+def get_api_key():
+    try:
+        config_file = os.path.expanduser('~/.certificator.cfg')
```

Comment:
it's not a good practice* to store sensitive information such as private API keys on text files. the best way to handle it is by using envirionment variables. for that I recommend using the lib [prettyconf](https://github.com/osantana/prettyconf/) and creating a `conf.py` file, that should look something like the snippet below.

```python

from prettyconf import config

class Settings:
    MEETUP_API_KEY = config("MEETUP_API_KEY", default="")


settings = Settings()

```

And by using this approach this function isn't necessary anymore.

\* I recommend reading the [12-factor app](https://12factor.net/) for good practices on building web applications. although this lib isn't a web application it's designed to support CLI, web apps, scripts etc.